Makefile

```
@@ -32,3 +32,12 @@ endif
 	cd $(tmpdir); zip -r ../$(artifact_name)-$(version).zip *
 	rm -rf $(tmpdir)
 
+.PHONY: package-ecs
+package:
```

Update the target name here:

```suggestion
package-ecs:
```